gcp-minimal/variables.tf

```diff
@@ -16,4 +16,4 @@ variable "zenml-version" {
   description = "The version of ZenML being used"
   default     = "0.12.0"
   type        = string
-}
\ No newline at end of file
+}
```

Comment:
Maybe we can remove this extra space too? Otherwise LGTM! thanks for adding the descriptions :)

Reply:
Not quite sure what you mean. Do you mean to remove the extra line at the end of the file? I think in general that's a good practice, but are you saying that `terraform` doesn't like that?

Reply:
My bad, I thought it was a space and not a line. Good to have it then, tf won't complain either